src/index.html

```diff
@@ -11,7 +11,176 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form class="form" action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label for="surname" class="label">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            class="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label for="name" class="label">
+          Name:
+          <input
+            type="text"
+            name="name"
+            id="name"
+            class="name"
+            autocomplete="off"
+          >
+        </label>
+        <label for="age" class="label">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            id="age"
+            class="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label for="dob" class="label">
+          Full date of birth:
+          <input
+            type="date"
+            name="dob"
+            class="dob"
+            id="dob"
```

Comment:
@elkhayate abbreviations and shorthands doesn't add up to code readability.
Better to have names longer, but clear and descriptive
```suggestion
            name="date-of-birth"
            class="date-of-birth"
            id="date-of-birth"
```